lib/CLIManager.js

```diff
@@ -276,6 +270,29 @@ class CLIManager {
     this._commandsManager.argv;
   }
 
+  /**
+   * Makes sure that if profile exists, token will be invalidated. If an error occurs, it logs but doesn't pass it to callback.
+   * @param {String} profileName
+   * @param done
+   * @returns {*}
+   * @private
+   */
+  _logoutUserFromProfile(profileName, done) {
+    const profile = this._setup.findProfileByName(profileName);
+    if (isNullOrUndefined(profile)) {
+      return setImmediate(() => { done(); });
```

Comment:
Maybe change this to setImmediate(done)
Also, shouldn't we also include the error?

Reply:
there would be no error in this case - no job to be done